### ISSUE TITLE
Additional content displayed on hover or focus is not dismissible (#8743440)

### ISSUE THREAD

**Issue by Shubham-Kanojiya** [Shubham-Kanojiya/Prodcheckgithub#5]
### Accessibility Issue Details

- **Issue ID**: 8743440
- **Severity**: Critical

### Page

- **Page**: example
- **URL**: https://example.com

### WCAG Information

- **Success Criteria**: 1.4.13
- **Level**: AA
- **Guidelines**: WCAG 2.2

### Recommendation

<p>Use JavaScript to provide a mechanism for dismissing the additional content on pressing an Esc key. This will ensure that users can dismiss the content without having to move their mouse pointer or keyboard focus. Alternatively, make sure that the additional content doesn't obscure other content on the page. One method of achieving is to display additional content on top of white space or over a decorative background image. </p>

### Actual Result Vs Expected Result

- **Actual Result**: <p>Additional content displayed when users hover over with a mouse or when their keyboard focus moves on bbbb and bbbb image buttons can't be dismissed. This made it difficult for screen magnifier users to access the content that is obscured by the additionally revealed content.</p>
- **Expected Result**: <p>Additional content displayed on mouse hover or keyboard focus should be dismissible. </p>

---
Created by A11yNow • Issue ID: 8743440
